Jenkinsfile_CNP

```diff
@@ -268,20 +268,20 @@ withPipeline(type, product, component) {
     // all the AAT tasks in the master pipeline finished, so we update the env variable to promote to prod tasks
   }
 
-  before('functionalTest:preview') {
-    yarnBuilder.yarn('yarn-update')
-    yarnBuilder.yarn('install-dependencies')
-    yarnBuilder.yarn('lint-ui-tests')
-  }
-
-  afterSuccess('functionalTest:preview') {
-    env.PARALLEL_CHUNKS = 12
-    yarnBuilder.yarn('test:functional')
-    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'output/**/*'
-    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/**/*'
-    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/serenity/**/*'
-    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test-results/functional/**/*'
-  }
+//  before('functionalTest:preview') {
```

Comment:
Should maybe add a short comment explaining why we're commenting these tests out

Reply:
Hey Qasim, my bad for raising PR early. Was commenting to test. As it's worked i've removed them entirely. Thanks for the comment though!

Will mention this in standup tomorrow and get everyones thoughts before continuing with it